packages/gatsby/src/utils/client-assets-for-template.ts

```diff
@@ -0,0 +1,153 @@
+import * as path from "path"
```

Comment:
Extracted to separate module from `packages/gatsby/src/utils/worker/child/render-html.ts`

Reply:
Just to clarify - we need to have it as a separate module because we re-use it later in bundle-webpack, right?

Reply:
Hmm, possibly we could revert this split because we don't use it in engine itself. But it's still beneficial because we don't have to load more things from workers code than needed (`child/render-html.ts` is not loaded by main process, just by html/pqr worker processes)